src/main/java/com/htisolutions/poolref/controllers/ResetPasswordController.java

```diff
@@ -0,0 +1,91 @@
+package com.htisolutions.poolref.controllers;
+
+import com.htisolutions.poolref.entities.User;
+import com.htisolutions.poolref.services.ResetPasswordService;
+import com.htisolutions.poolref.viewModels.ResetPasswordNicknameViewModel;
+import com.htisolutions.poolref.viewModels.ResetPasswordPasswordViewModel;
+import com.htisolutions.poolref.viewModels.ResetPasswordQuestionViewModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.*;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+@RequestMapping("/reset-password")
+public class ResetPasswordController {
+
+    private ResetPasswordService resetPasswordService;
+    private User user;
```

Comment:
Being global ensures that us has to get set in a certain way so that the function can't be called by anyone to change the password of a certain user